src/components/Pollquote/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+
+interface PullQuoteProps {
+  //     /** Text for the poll quote */
+  text: string
+  //     /** Custom css for poll quote */
+  style?: string
+}
+
+export default function PullQuote(props: PullQuoteProps) {
+  const pullQuoteStyle = css`
+    display: flex;
+    ${props.style};
+  `
+  return (
+    <div className={pullQuoteStyle}>
+      <svg width="60" height="36">
```

Comment:
Also let's not use an svg here, let's use a div or span with css for the positioning and size. that way the quote will match the font of the text.

Reply:
will do